src/commands/tty.ts

```diff
@@ -4,24 +4,28 @@ const HIDE = "?25l";
 const SHOW = "?25h";
 const CLEAR_LINE = "2K";
 
-const enc = new TextEncoder();
+export const isatty = Deno.isatty(Deno.stdout.rid);
 
-export function goUp(rows: number) {
-  writeCommand(rows + UP);
-}
+export class TTY {
```

Comment:
Not sure what I have to think about this. Yes, it's nice that we deal with an actual injected class for unit testing, but I'm not sure if it's really beneficial for some utility class that takes care of command line stuff. But I assume you made this decision with reason :-)

Reply:
well... its a long story :D 
I first tried to go with the suggestion in the ticket where Johannes said just NOOP the calls in these tty calls. Implemented it and because of that turned the TTY in a class. But then I realized, hey wait a sec, we are also checking for TTY on different occasions e.g. inside decorators and then sometimes also no-oping. Which I thought is an antipattern. Why have a wrapper when all it does is nooping back to the underling wrapped unit? I also remembered a book text that said: if you have a flag inside your code that branches code that's probably a sign of smelly code.
So I decided it makes more sense to streamline the behavior of wrapper without this branching flag and instead put that logic into the factory. She should ultimately decide (depending on the flag) if the unit needs to be decorated (and thus the behavior altered or not). So that lead me to the refactoring of the whole code.
In the end the TTY class lost their flag control behavior because now its depending if the code is decorated or not. However I did not revert my TTY class refactoring as it might have slight advantages when unit testing the code so we can insert mocks or spies.... its however not super critical and was not the main design design. It was more the laziness to revert the already done changes... :D